knowledge-base/grid-csv-export-change-field-delimiter.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Grid export to CSV - change the default field delimiter (comma)
+description: How to export the Grid to CSV with a different/custom field delimiter?
+type: how-to
+page_title: Grid export to CSV - change the default field delimiter (comma)
+slug: grid-kb-csv-export-change-field-delimiter
+position: 
+tags: grid, export, csv, field, delimiter, change, semicolon, comma, custom
+ticketid: 1576816
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+I want to change the default field delimiter when exporting the Grid to CSV file. How to set a different list separator?
+
+I want to use a semicolon field delimiter for the exported CSV Grid instead of comma. How to achieve this?
+
+>tip At the time of writing (**UI for Blazor 3.5.0**), the Blazor Grid [does not expose a built-in option for configuring the field delimiter](https://feedback.telerik.com/blazor/1577167-option-to-configure-the-field-delimiter-when-exporting-to-csv). You may vote for the enhancement and follow it to get status email updates.
```

Comment:
Maybe skip the email part, as it can sound spammy :)

```suggestion
>tip At the time of writing (**UI for Blazor 3.5.0**), the Blazor Grid [does not expose a built-in option for configuring the field delimiter](https://feedback.telerik.com/blazor/1577167-option-to-configure-the-field-delimiter-when-exporting-to-csv). You may vote for the enhancement and follow it to receive status updates.
```